include/math/Math.h

```diff
@@ -0,0 +1,45 @@
+#pragma once
+
+#include <glm/mat2x2.hpp>
+#include <glm/mat3x3.hpp>
+#include <glm/mat4x4.hpp>
+#include <glm/vec2.hpp>
+#include <glm/vec3.hpp>
+#include <glm/vec4.hpp>
+
+namespace arch::math {
+
+using Byte = std::byte;
+
+using i8 = int8_t;
+using i16 = int16_t;
+using i32 = int32_t;
+using i64 = int64_t;
+
+using u8 = uint8_t;
+using u16 = uint16_t;
+using u32 = uint32_t;
+using u64 = uint64_t;
+
+using f32 = float_t;
+using f64 = double_t;
```

Comment:
maybe some alias for long double?

Reply:
Problem with it, is that C++20 doesn't defines bit size for `long double`. To do so we would need C++23.
[Fixed width floating-point types](https://en.cppreference.com/w/cpp/types/floating-point)

Reply:
Maybe at least some shortcut for long double then? just for the sake of having nice aliases for most of the types

Reply:
so what you propose as an alias?

Reply:
I think simply first letters of long double -> ld